Pawn.lua

```diff
@@ -780,6 +786,25 @@ function PawnLinkWranglerOnTooltip(Tooltip, ItemLink)
 	PawnAttachIconToTooltip(Tooltip, false, ItemLink)
 end
 
+-- ArkInventory Rule
+function ArkInventoryPawnUpgradeRule( ... )
+    local fn = "PAWNUPGRADE" -- Rule name for errors
+    if not PawnIsInitialized then VgerCore.Fail("Can't check to see if items are upgrades until Pawn is initialized") return end
+
+    -- Verify that the item string information is loaded and not nil and that it is a valid item before continuing
+    if not ArkInventoryRules.Object.h or ArkInventoryRules.Object.class ~= "item" then return false end
+
+    -- Parse the incoming item and retrieve the data
+    local ac = select( '#', ... )
```

Comment:
Interesting; never seen that `select("#", ...)` syntax before. It looks like you don't use this value anymore, though?

Reply:
Yeah, I've never seen it either, but I found it in other ArkInventory rules. I'm not entirely sure what the connection here is between the incoming parameters and the rest of the AI item code, but I'm hesitant to remove it without fully understanding what it's up to when it's in all the other rules.

Reply:
It works without it, though, so I'll remove it. No need to carry crusty code forward.

Reply:
I'm guessing that `ac` stands for "argument count" and `select("#", ...)` is getting the number of items in `...`, like how you can say `#mytable` to get the number of items in `mytable`. So based on that completely-made-up explanation, it seems entirely safe to remove.  _**ENGINEERING!**_

Reply:
If it still works after you delete it, you're good to go haha